src/modules/compliance/src/lib/procedures/EnsureSshdOption.cpp

```diff
@@ -9,34 +9,10 @@
 
 namespace compliance
 {
-AUDIT_FN(EnsureSshdOption, "optionName:Name of the SSH daemon option:M", "optionRegex:Regex that the option value has to match:M")
+namespace
+{
+Result<std::map<std::string, std::string>> GetSshdOptions(ContextInterface& context)
 {
-    auto log = context.GetLogHandle();
-
-    auto it = args.find("optionName");
-    if (it == args.end())
-    {
-        return Error("Missing 'optionName' parameter", EINVAL);
-    }
-    auto optionName = std::move(it->second);
-
-    it = args.find("optionRegex");
-    if (it == args.end())
-    {
-        return Error("Missing 'optionRegex' parameter", EINVAL);
-    }
-    auto optionRegex = std::move(it->second);
-    regex valueRegex;
-    try
-    {
-        valueRegex = regex(optionRegex);
-    }
-    catch (const regex_error& e)
-    {
-        OsConfigLogError(log, "Regex error: %s", e.what());
-        return Error("Failed to compile regex '" + optionRegex + "' error: " + e.what(), EINVAL);
-    }
-
     auto sshdTestOutput = context.ExecuteCommand("sshd -T 2>&1");
```

Comment:
>auto sshdTestOutput = context.ExecuteCommand("sshd -T 2>&1"); [](http://example.com/codeflow?start=4&length=61)

Why are we re-implementing this instead of reusing the commonutils SSH calls?